src/arduino/buzzer/buzzer.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Buzzer
+level: 1
+author: Adrian Helle
+---
+
+# Introduksjon {.intro}
+
+Nå som vi har sett litt på hvordan vi kan kontrollere en lysdiode på forskjellige måter, kan
+gå over til å lage litt lyd! 
+
+# Steg 1: Finn frem utstyr {.activity}
+
+Før vi kan begynne å lage kretsen og koden, er vi nødt til å finne frem alt vi trenger.
+I denne oppgaven trenger vi i hovedsak en buzzer.
+
+## Dette trenger du {.check}
+
++ 1 Arduino Uno
++ 1 breadboard
++ 2 ledninger
++ 1 buzzer
+
+<figure><img src="buz.jpg" style="width: 900px"></figure>
+
+# Steg 2: Lag kretsen {.activity}
+
+Før vi kan begynne med programmeringen, så er vi nødt til å lage en krets.
+Hvis du nå har funnet frem alt du trenger kan du følge instruksjonene og illustrasjonen
+under
+
+<figure><img src="krets.png" style="width: 900px"></figure>
+
+
+## Sjekkliste {.check}
+
++ Ledning fra GND på Arduinoen til den negative lederen på buzzeren
++ Ledning fra 11~ på Arduinoen til den positive lederen på buzzeren
+
+## Porter merket med ~ {.protip}
+
+Husker du hva "~" betyde? Det har seg slik at alle porter merket med "~"
+har en spesiel funksjon kalt PWM. Dette står for "Pulse With Modulation"
+og brukes for å lage et analogt signal. I denne oppgaven bruker vi dette
+for å generer toner.
+
+# Steg 3: Lag en tone {.activity}
+
+*Nå skal vi lage vår første tone med Arduinoen.*
+
+## Sjekkliste {.check}
+
++ Åpne Arduino-programmet om det ikke allerede er åpent.
++ Arduino-programmet starter med denne koden:
+
+  ```cpp
+  void setup(){
+
+  }
+
+  void loop(){
+
+  }
+  ```
+
++ Skriv denne koden:
+
+  ```cpp
+  int lyd = 11;
+
+  void setup(){
+    pinMode(lyd, OUTPUT);
+  }
+
+  void loop(){
+    tone(lyd, 880);
+  }
+  ```
+
++ Trykk på ![](upload.png) for å laste opp koden. Denne sjekker først om koden
+er riktig, og så vil programmet ditt kjøre på arduinoen.
++ Lager den lyd?
+
+### Virker det ikke? {.protip}
+
+Hvis det ikke virker, så kan det hende at Arduino-programmet står på feil __port__
+og/eller __brett__. Da kan du sjekke disse to tingene:
+
++ Brett er satt riktig: __Tools -> Board -> Arduino/Genuino Uno__
+
++ Port er satt riktig:
+	+ Windows: __Tools -> Port -> COM1__ (kan være et annet tall)
+	![](port.png)
+	+ Mac: __Tools -> Port ->/dev/tty.usbmodem262471__ (kan være et annet tall)
++ Lysdioden er koblet riktig vei
+	+ Den korteste "foten" skal gå til GND
+
+Hvis dette ikke fungerer, kan du prøve å lukke programmet og åpne det igjen.
+
+
+### Utfordringer {.challenge}
+
++ Klarer du å finne ut hvilken tone dette er? (Hint: 880 er frekvensen til tonen)
++ Klarer du å lage en annen tone?
+
+# Steg 4: Bruk en fotoresistor til å styre lyden {.activity}
+
+Vi kan bruke "analog input" for å styre lyden også! La oss se 
+hva som skjer hvis vi gjør slik at en fotoresistor (motstanden
+variere ettersom hvor mye lys den får inn) styrer lyden.
```

Comment:
Vanskelig ordlyd på den synes jeg. Kanskje skrive om til noe ala:

Vi kan bruke en annen sensor for å styre lyden også. La oss se hvordan vi kan få en fotoresistor til å styre lyden. En fotoresistor varierer motstanden etter hvor mye lys den får inn.

?